### DISCUSSION TITLE
How to Cleanly Uninstall?

### DISCUSSION THREAD

**Discussion by Jaballadares** [johnlindquist/kit#372] [Q&A]
Hello! Kit is amazing, I downloaded Kit but unfortunately cannot use it on my work machine. Are there any recommendations for cleanly uninstalling?

I've already removed `/Applications/Kit.app` as well as the `~/.kenv` and `~/.kit`files. Should I be all set?

Thank you!

**Comment by johnlindquist** [comment]
Yup, that's all. 👍

**Comment by o-az** [comment]
Adding uninstallation instructions in the official docs would be great.

**Reply by johnlindquist** [reply]
Will do. For now, here you go:

1. Delete Kit.app
2. Delete 

~/.kit
~/.kenv
~/.knode

**Reply by alexilyaev** [reply]
I couldn't find the uninstall instructions in the docs
